specification/api-correlationcontext.md

```diff
@@ -1,24 +1,27 @@
-# DistributedContext API
+# Correlations API
 
-An `Entry` is used to label anything that is associated
-with a specific operation, such as an HTTP request.
+An `Entry` is used to label anything that is associated with a specific
+operation, such as an HTTP request.
 
-`DistributedContext` is an abstract data type that represents a collection of entries.
-Each key of `DistributedContext` is associated with exactly one value. `DistributedContext` is serializable,
-to facilitate propagating it not only inside the process but also across process boundaries.
-`DistributedContext` is used to annotate telemetry with the name:value pair `Entry`.
-Those values can be used to add dimension to the metric or additional context properties to logs and traces.
+`CorrelationContext` is an abstract data type that represents a collection of
+entries. Each key of `CorrelationContext` is associated with exactly one value.
+`CorrelationContext` is serializable, to facilitate propagating it not only
+inside the process but also across process boundaries. `CorrelationContext` is
+used to annotate telemetry with the name:value pair `Entry`. Those values can
+be used to add dimension to the metric or additional context properties to logs
```

Comment:
This last statement is not very useful (it's not your fault). I think we want to work on some clarity here, but not as part of the same PR. We discussed in ~2 weeks ago's Spec SIG how this has been requested and underspecified. I think it should eventually read that:

"Name/value correlations are logically applied to trace and metric events. SDKs SHOULD expose the correlation context of a span associated with its start event by including correlations in the Span attributes. SDKs SHOULD expose the correlation context associated with metric events by including correlations as metric labels." Let's put this change off, though.

Reply:
I also suggest we don't clarify too many things as part of this PR, and open a new ticket for tracking this.